src/PrizeVault.sol

```diff
@@ -496,26 +528,39 @@ contract PrizeVault is TwabERC20, Claimable, IERC4626, ILiquidationSource, Ownab
         }
     }
 
+    /**
+     * @notice Transfers yield fee shares to the yield fee recipient
+     * @dev Will revert if the caller is not the yield fee recipient or if zero shares are withdrawn
+     */
+    function claimYieldFeeShares(uint256 _shares) external onlyYieldFeeRecipient {
+        if (_shares == 0) revert MintZeroShares();
+
+        uint256 _yieldFeeBalance = yieldFeeBalance;
+        if (_shares > _yieldFeeBalance) revert SharesExceedsYieldFeeBalance(_shares, _yieldFeeBalance);
+
+        yieldFeeBalance -= _yieldFeeBalance;
+        _mint(yieldFeeRecipient, _shares);
+    }
+
     /* ============ LiquidationSource Functions ============ */
 
     /// @inheritdoc ILiquidationSource
-    /// @dev Returns the withdrawable amount of the available yield minus any yield fees.
+    /// @dev Returns the liquid amount of `_tokenOut` minus any yield fees.
     /// @dev Supports the liquidation of either assets or prize vault shares.
-    function liquidatableBalanceOf(address _token) public view returns (uint256) {
+    function liquidatableBalanceOf(address _tokenOut) public view returns (uint256) {
         uint256 _totalSupply = totalSupply();
-        uint256 _maxLiquidation;
-        if (_token == address(this)) {
-            // Liquidation of vault shares is capped to the TWAB supply cap (max uint96).
-            _maxLiquidation = type(uint96).max - _totalSupply;
-        } else if (_token == address(_asset)) {
+        uint256 _maxAmountOut;
+        if (_tokenOut == address(this)) {
+            // Liquidation of vault shares is capped to the TWAB supply limit.
+            _maxAmountOut = _twabSupplyLimit(_totalSupply);
+        } else if (_tokenOut == address(_asset)) {
             // Liquidation of yield assets is capped at the max withdraw.
-            _maxLiquidation = yieldVault.maxWithdraw(address(this)) + _asset.balanceOf(address(this));
+            _maxAmountOut = yieldVault.maxWithdraw(address(this)) + _asset.balanceOf(address(this));
         } else {
             return 0;
         }
-        uint256 _availableYield = _availableYieldBalance(totalAssets(), _totalSupply);
-        uint256 _liquidYield = _availableYield >= _maxLiquidation ? _maxLiquidation : _availableYield;
-        return _liquidYield.mulDiv(FEE_PRECISION - yieldFeePercentage, FEE_PRECISION);
+        uint256 _liquidYield = _availableYieldBalance(totalAssets(), _totalSupply).mulDiv(FEE_PRECISION - yieldFeePercentage, FEE_PRECISION);
```

Comment:
A comment explaining this line would be helpful

Reply:
What is the rounding direction here?  Should we make it explicit?

Reply:
It rounds down. I don't know of any math libraries that round up by default in integer division, so I doubt auditors will assume that behavior.

Perhaps we can just specify that it rounds down in the comments to address any confusion.